src/isomorphic/modern/element/__tests__/ReactJSXElementValidator-test.js

```diff
@@ -156,15 +187,20 @@ describe('ReactJSXElementValidator', () => {
     };
     iterable.entries = iterable['@@iterator'];
 
-    void <Component>{iterable}</Component>;
+    ReactTestUtils.renderIntoDocument(<Component>{iterable}</Component>);
 
     expectDev(console.error.calls.count()).toBe(0);
   });
 
   it('does not warn when the element is directly as children', () => {
     spyOn(console, 'error');
 
-    void <Component><Component /><Component /></Component>;
+    void (
```

Comment:
Let's change to `TestUtils.renderIntoDocument` too for consistency?

Reply:
Will do :) The reason I didn't add `TestUtils.renderIntoDocument` here was because it's only necessary to fire key warnings, which this test wasn't testing.